services/ui-src/src/components/reports/ReportProvider.tsx

```diff
@@ -16,7 +17,7 @@ import {
 import { reportErrors } from "verbiage/errors";
 
 export const ReportContext = createContext<ReportContextShape>({
-  reportStatus: "",
+  reportStatus: {} as AnyObject,
```

Comment:
As discussed, this may be changing soon but it's fine for right now. We'll have to look into exactly what we want the data structure of this to be.